src/render_tasks/d3d12_rtao_task.hpp

```diff
@@ -127,9 +114,9 @@ namespace wr
 			{
 				auto& as_build_data = fg.GetPredecessorData<wr::ASBuildData>();
 
-				data.out_uav_from_rtv = std::move(as_build_data.out_allocator->Allocate());
-				data.in_gbuffers = std::move(as_build_data.out_allocator->Allocate());
-				data.in_depthbuffer = std::move(as_build_data.out_allocator->Allocate());
+				data.out_uav_from_rtv = std::move(as_build_data.out_allocator->Allocate(1));
```

Comment:
No reason to explicitly allocate 1, allocate() does the same.

Reply:
readability